### PR TITLE
Cleanup for 4.12

### PR DESCRIPTION
**Pull request by shubhamkumar13**
This PR aims to cleanup all the `4.12.*` variants from the scripts and documentation

### CODE REVIEW

File: ocaml-versions/README.md [hunk 1]
```diff
@@ -16,10 +14,11 @@ following versions:
 
 1. Serial + Throughput
   + `4.12.0+stock.json`
```
Comment (shakthimaan) [inline ocaml-versions/README.md:16]: 4.12.0 entries are still there in the README?

Comment (shubhamkumar13) [reply ocaml-versions/README.md]: Nice catch!

**Comment by shubhamkumar13** [comment]
Thanks @shakthimaan for the review!
I've updated [`ocaml-versions/README.md`](https://github.com/ocaml-bench/sandmark/blob/master/ocaml-versions/README.md) with the statements that I've missed

File: ocaml-versions/README.md [hunk 2]
```diff
@@ -4,23 +4,19 @@
...
-  + `4.14.0+trunk.json`
+  + `5.00.0+trunk.json`
 2. Serial + Latency
   + `4.12.0+stock+instrumented.json` -- compiled with tracing on, which slows down the execution.
 3. Parallel + Throughput.
-  + `4.12.0+domains.json`
-  + `4.12.0+domains+effects.json`
+  + `4.14.0+domains.json`
 4. Parallel + Latency.
   + `4.12.0+domains+effects+instrumented.json` -- compiled with tracing on, which slows down the execution.
```
Comment (shakthimaan) [inline ocaml-versions/README.md:22]: Is this required?

Comment (shubhamkumar13) [reply ocaml-versions/README.md:22]: I've opened an issue https://github.com/ocaml-bench/sandmark/issues/266, as soon as this gets resolved we can update instrumented pausetimes

**Pull request merged**